src/components/EventList.spec.tsx

```diff
@@ -46,33 +63,67 @@ describe('EventList', () => {
     });
 
     it('contains a sign-up button for each event', () => {
+        const spyOnSubmit = cy.spy();
+
         mountWithProviders(
-            <EventList events={ dummyEvents } org={ dummyOrg }/>,
+            <EventList
+                eventResponses={ dummyEventResponses }
+                events={ dummyEvents }
+                onEventResponse={ spyOnSubmit }
+                org={ dummyOrg }
+            />,
         );
 
-        cy.contains('misc.eventList.signup');
+        //Checks for buttons on all events
+        cy.findByText('misc.eventList.signup');
```

Comment:
What is the purpose of this?

Reply:
I see now it's unnecessary since line 81 is already doing the same thing. Will remove this!